### PR TITLE
Add template for proposals

### PR DESCRIPTION
**Pull request by jdolitsky**
As discussed in this week's meeting, add a template for people to use to submit proposals so that they are more accessible for people to compare.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,10 @@
 
 Propose how to describe and query relationships between objects stored in an OCI registry.
 
-## Proposal
+## Background
```
Comment (jdolitsky) [inline README.md:7]: Changed this to "Background" so as not to confuse with the proposals themselves

**Comment by nishakm** [comment]
#13 creates a folder called `proposals` and perhaps that's where these changes should land with a link in the existing README.md

File: README.md [hunk 2]
```diff
@@ -15,13 +16,30 @@ Link to [proposal](https://github.com/opencontainers/tob/blob/main/proposals/wg-
...
+
+The following proposals are currently being evaluated by the WG:
+
+| ID | Link                              | Description            |
+| -- | --------------------------------- | ---------------------- |
+| A  | [View](./proposals/PROPOSAL_A.md) | *Reserved for Steve*   |
+| B  | [View](./proposals/PROPOSAL_B.md) | *Reserved for Dan*     |
+| C  | [View](./proposals/PROPOSAL_C.md) | *Reserved for Nisha*   |
+| D  | [View](./proposals/PROPOSAL_D.md) | *Reserved for Brandon* |
+
```
Comment (jdolitsky) [inline README.md:29]: @nishakm yes, my thoughts exactly!

**Comment by lachie83** [comment]
Thanks for putting this together @jdolitsky. LGTM

**Review by lachie83 — APPROVED** [review]

**Pull request merged**